### PR TITLE
Fix tob loottracker

### PR DESCRIPTION
**Pull request by pklite**
this will fix* the tob loots not being recorded by loot tracker

<sup><sub>* = probably? idk cant test it, im no pvmer </sup></sub>

### CODE REVIEW

**Comment by zeruth** [comment]
Can we get someone to test this?

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java [hunk 1]
```diff
@@ -516,16 +515,16 @@ private void onWidgetLoaded(WidgetLoaded event)
...
 				break;
-			case (WidgetID.THEATRE_OF_BLOOD_GROUP_ID):
+			case (WidgetID.THEATRE_OF_BLOOD_REWARD_GROUP_ID):
 				if (chestLooted)
 				{
 					return;
 				}
-				int region = WorldPoint.fromLocalInstance(client, client.getLocalPlayer().getLocalLocation()).getRegionID();
-				if (region != THEATRE_OF_BLOOD_REGION)
+				if (!(client.getLocalPlayer().getWorldLocation().getRegionID() == THEATRE_OF_BLOOD_REGION))
```
Comment (ThatGamerBlue) [inline runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java]: `if (getRegionID() != THEATRE_OF_BLOOD_REGION)`

Comment (pklite) [reply runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java]: @ThatGamerBlue is that what you meant? still havent gotten intellij to stop using gradle 5 so edited directly in github

**Comment by ThatGamerBlue** [comment]
Looks good to me, waiting for a tester then I'll approve it.

**Comment by HSJ-OSRS** [comment]
case (WidgetID.THEATRE_OF_BLOOD_REWARD_GROUP_ID): fixes it the other change breaks it again

**Comment by pklite** [comment]
> case (WidgetID.THEATRE_OF_BLOOD_REWARD_GROUP_ID): fixes it the other change breaks it again

the 

> case (WidgetID.THEATRE_OF_BLOOD_REWARD_GROUP_ID): fixes it the other change breaks it again

Oh yeah, i just realized that the loot place is in an instance. Did you get to test it with the first change?

**Comment by HSJ-OSRS** [comment]
yeah i tried it with just the widget id change and it works

**Pull request merged**